data/Localization/german_(germany)/global.ini

```diff
@@ -39707,79 +39707,79 @@ ServiceBeacon_Revive_Obj_Title=Rescue ~serviceBeacon(InitiatorName)
 ShipSelector_Header_Deliver=Anfrage zur Fahrzeugauslieferung
-ShipSelector_Header_Impounded=Beschlagnahmtes Fahrzeug reklamieren
-ShipSelector_Header_Main=Fuhrparkleiter
-ShipSelector_Header_RetrievalFailed=Fahrzeug nicht zurückerhalten
-ShipSelector_Header_Retrieved=Erfolgreich abgerufenes Fahrzeug
+ShipSelector_Header_Impounded=Beschlagnahmtes Fahrzeug ausbezahlen
+ShipSelector_Header_Main=Flottenmanager
+ShipSelector_Header_RetrievalFailed=Beanspruchung fehlgeschlagen
+ShipSelector_Header_Retrieved=Erfolgreich ausgeparkt
 ShipSelector_Header_Retrieving=Fahrzeug abrufen...
 ShipSelector_Header_VehicleInQueue=Fahrzeug in der Warteschlange
 ShipSelector_IdleScreen_Title=Konsole für die Fahrzeugrückholung
-ShipSelector_ImpoundingTime=Dauer der Beschlagnahme
-ShipSelector_InsufficientCredits=UNZUREICHENDE KREDITE
+ShipSelector_ImpoundingTime=Dauer der Beschlagnahmung
+ShipSelector_InsufficientCredits=UNZUREICHENDE UEC
 ShipSelector_Label_PayFine=Geldstrafe zahlen
-ShipSelector_Label_Relocate=Verlegen Sie
-ShipSelector_Label_Retrieve=abrufen.
-ShipSelector_Label_Store=Blind
+ShipSelector_Label_Relocate=Umziehen
+ShipSelector_Label_Retrieve=Ausparken
+ShipSelector_Label_Store=Einparken
 ShipSelector_Location=Standort
 ShipSelector_NoValidAreaSize=DIESE FAHRZEUGGRÖSSE KANN AN DIESEM STANDORT NICHT ANGEFORDERT WERDEN
 ShipSelector_NoValidAreaType=DIESER FAHRZEUGTYP KANN AN DIESEM STANDORT NICHT ANGEFORDERT WERDEN
 ShipSelector_PayDeductibleDescription=Sie können eine Selbstbeteiligung zahlen, um die Bearbeitungszeit des Antrags zu beschleunigen.
-ShipSelector_PayExpeditedDescription=Sie können eine optionale Gebühr entrichten, um die Bearbeitung des Antrags zu beschleunigen.
-ShipSelector_PayFineDescription=Dieses Fahrzeug ist beschlagnahmt worden. Um es freizugeben, muss eine Geldstrafe bezahlt werden.
-ShipSelector_PayPremium=PRÄMIENZAHLUNG
-ShipSelector_PositionInQueue=Position in Schwanz:
-ShipSelector_PoweredBy=EILDIENST POWERED BY:
-ShipSelector_RentalExpired=EXPIRED
-ShipSelector_Retrieving_Description=Bitte warten Sie, während Ihr Fahrzeug ausgeliefert wird.
+ShipSelector_PayExpeditedDescription=Sie können eine optionale Gebühr bezahlen, um die Bearbeitung des Antrags zu beschleunigen.
+ShipSelector_PayFineDescription=Dieses Fahrzeug ist beschlagnahmt worden. Um es freizugeben, wird eine Geldstrafe fällig.
+ShipSelector_PayPremium=PRÄMIE ZAHLEN
+ShipSelector_PositionInQueue=Position in Warteschlange:
+ShipSelector_PoweredBy=BESCHLEUNIGTER SERVICE, POWERED BY:
+ShipSelector_RentalExpired=ABGELAUFEN
+ShipSelector_Retrieving_Description=Bitte warten Sie, während Ihr Fahrzeug ausgeparkt wird.
 ShipSelector_Status=Status
 ShipSelector_StatusNotApplicable=--
 ShipSelector_Vehicle=Fahrzeug
 ShipSelector_VehicleDelivered=Geliefert
-ShipSelector_WaitingForService=Bereithalten, um die Liste der verfügbaren Fahrzeuge aufzurufen.
+ShipSelector_WaitingForService=Liste iherer Schiffe wird aufzurufen...
```

Comment:
ShipSelector_WaitingForService=Bitte warten, verfügbare Fahrzeuge werden abgerufen.
or
ShipSelector_WaitingForService=Liste Ihrer Schiffe wird abgerufen...
or
ShipSelector_WaitingForService=Liste Ihrer Schiffe wird aufgerufen...

Reply:
"iherer" is still wrong xD

Reply:
I have no idea what was going on, I somehow misread it myself.🤣